src/main/java/com/whalespottingjava/services/SightingService.java

```diff
@@ -35,4 +35,6 @@ public List<Sighting> getAllApprovedSightings() { // TODO: 12/10/2023 Returns ra
     public List<Sighting> getAllPendingSightings() {
         return this.sightingRepository.findByApprovedFalse();
     }
+
+    public void deleteRejectedPendingSighting(Long sightingId) { sightingRepository.deleteById(sightingId); }
```

Comment:
Even though this line fits as a one-liner, let's just split it out over 3 lines to match the styling of the methods above